modules/test/playwright/tests/notification-web/emailNotificationTemplate.spec.ts

```diff
@@ -271,3 +271,43 @@ test('can see all roles groups in email notification template recipients', async
 		emailNotificationTemplatePage.organizationRolesGroupTitle
 	).toBeVisible();
 });
+
+test('can see cc/bcc fields in UI when creating notification via API without passing them', async ({
```

Comment:
i think this description is not very clear, it could be something like `'can render cc/bcc fields in UI  without values when creating notification via API'`